Lib/test/test_faulthandler.py

```diff
@@ -93,18 +93,20 @@ def check_error(self, code, lineno, fatal_error, *,
                     fd=None, know_current_thread=True,
                     py_fatal_error=False,
                     garbage_collecting=False,
+                    c_stack=True,
```

Comment:
We probably want tests for:
1. `c_stack=False`
2. checking for "cannot get C stack..." on unsupported systems
3. `dump_c_stack`

Reply:
> checking for "cannot get C stack..." on unsupported systems

That's checked by the regex, there's a case for `<whatever>` messages in there.